1.1.0 build

xylar opened the pull request:
needed for apache-airflow

implementing pin_compatible to resolve overdepending

<!--
Thank you for pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

closes #9

BrentDorsey committed:
1.1.0 build

needed for apache-airflow

implementing pin_compatible to resolve overdepending

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found some lint.

Here's what I've got...


For **recipe**:

 * license_file entry is missing, but is required.

xylar committed:
Add LICENSE back

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

xylar commented:
@conda-forge-admin, please rerender

Commit:
MNT: Re-rendered with conda-build 3.19.2, conda-smithy 3.7.4, and conda-forge-pinning 2020.07.21.17.10.15

xylar commented on recipe/meta.yaml:
```diff
@@ -8,39 +8,92 @@ package:
   recipe-maintainers:
     - pmlandwehr
     - xylar
+    - BrentDorsey
```
@BrentDorsey, I rebased #9 and made some small changes (adding back the license file, mostly).

Could you confirm that you're still willing to be added as a maintainer?

xylar replied:
On second thought, since this is getting merged to a side branch, `1.1.0`, it's not the right place to add you.  I will ask in #10 instead.

xylar committed:
Update dependencies

xylar commented:
@conda-forge-admin, please rerender

xylar committed:
Fix hash

xylar committed:
Remove BrentDorsey as maintainer here

This may cause issues because it's getting merged to a different
branch.  I will add him in a different PR instead.

xylar commented:
@conda-forge-admin, please rerender

Commit:
MNT: Re-rendered with conda-build 3.19.2, conda-smithy 3.7.4, and conda-forge-pinning 2020.07.21.17.10.15
